test_scripts/Resumption/InteriorVehicleData/OnHashChange/002_OnHashChange_by_module_subscription_with_moduleId.lua

```diff
@@ -35,7 +34,7 @@ common.Step("App activation", common.activateApp)
 common.Title("Test")
 for _, moduleName in pairs(common.modules)do
   common.Step("OnHashChange after adding subscription to " .. moduleName, common.GetInteriorVehicleData,
-    { moduleName, common.getModuleId(moduleName, testModuleNumber), isSubscribed, isNotCached, expectNotif })
+    { moduleName, common.getModuleId(moduleName, 2), isSubscribed, isNotCached, expectNotif })
```

Comment:
`testModuleNumber` can be removed also from 005, 010, 026

Reply:
Variable `testModuleNumber` is used in mentioned scripts because it is used multiple times in the script. That is why the variable remains.